app/controllers/questions_controller.rb

```diff
@@ -17,6 +17,8 @@ def create
 
     def show
         @question = Question.find(params[:id])
+        @solution = Solution.new
+        @solutions = Solution.where(question_id: @question.id)
```

Comment:
まずはモデルの修正からですね。

- app/models/question.rb

```
  has_many :solutions
```

修正後なら，次のようにかけます

```suggestion
        @solutions = @question.solutions
```